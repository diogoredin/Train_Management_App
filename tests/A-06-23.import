SERVICE|590|12.2|07:02|A1|07:03|A2|07:04|B3|07:26|A4|07:48|A5|07:57|A6|08:01|A7|08:06|A8|08:10|A9
SERVICE|690|12.2|07:06|B1|07:17|B2|07:31|B3|07:53|B4|08:15|B5|08:24|B6|08:28|B7
SERVICE|670|12.2|07:06|B1|07:17|B2|07:31|B3|07:54|A4|08:14|B5|08:24|B6|08:25|B7
SERVICE|680|12.2|07:06|B1|07:17|B2|07:31|B31|07:52|B4|08:14|A5|08:16|B6|08:18|B7
SERVICE|780|12.2|07:06|B1|07:17|B2|07:31|B3|07:55|B4|08:15|A5|08:25|B6|08:38|B7
PASSENGER|Azura
PASSENGER|Boethiah
PASSENGER|Clavicus Vile
PASSENGER|Hermaeus Mora
PASSENGER|Hircine
PASSENGER|Malacath
PASSENGER|Mehrunes Dagon
PASSENGER|Mephala
PASSENGER|Meridia
