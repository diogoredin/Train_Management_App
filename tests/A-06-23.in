6
3
1
A2
B7
2017-12-31
07:00
0
0
0
