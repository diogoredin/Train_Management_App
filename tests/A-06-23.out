Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-31 @ 8.75
Serviço #590 @ 8.07
07:03 A2
07:04 B3
07:26 A4
07:48 A5
Serviço #680 @ 0.68
08:14 A5
08:16 B6
08:18 B7
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 